README.md

```diff
@@ -11,25 +11,32 @@ Your personal practice companion
 </p>
 
 <br/>
+
+<p align="center">
+<a href='https://play.google.com/store/apps/details?id=app.musikus&hl=de&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'>
+  <img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png'height="70"/>
+</a>
+</p>
+
 <div align="center">
   <a href="https://github.com/matthiasemde/musikus-android/actions/workflows/ci.yml"><img src="https://github.com/matthiasemde/musikus-android/actions/workflows/ci.yml/badge.svg" alt="CI"></a>
   <a href="https://github.com/matthiasemde/musikus-android/releases/latest"><img src="https://img.shields.io/github/v/release/matthiasemde/musikus-android" alt="release"></a>
   <a href="https://github.com/matthiasemde/musikus-android/releases"><img src="https://img.shields.io/github/v/release/matthiasemde/musikus-android?include_prereleases" alt="nighlty-release"></a>
 </div>
 <br/>
 
+
 ## 🧪 Musikus is in alpha
 
 ### Join
-If you want to join our group of testers, write us an email to [alpha@musikus.app](mailto:alpha@musikus.app) using your Gmail-account and we will add you to the list.
+
+If you want to join our group of testers, write us an email to [alpha@musikus.app](mailto:alpha@musikus.app) using your Google account and we will add you to the list. If you are amongst the list of testers, the badge above will bring you to the Play Store listing.
+Alternatively, you can always download the latest release [from GitHub](https://github.com/matthiasemde/musikus-android/releases/latest).
```

Comment:
People _can_ download the latest release from GitHub, but then they are not really joining the alpha. The alpha is happening on GooglePlay.

Reply:
Okay, fair point. But I would like to encourage people to test the app even though they might not want to be an "official alpha tester".
It's just about mentioning that our releases are also uploaded to GitHub.

Maybe we can rephrase it to something like "Pre-built release apks can be found on the release page".